nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -726,6 +726,15 @@
     dirs:
       - 'resources'
 
+- module-name: 'cpuinfo.cpuinfo' # checksum: fc219e6
+  anti-bloat:
+    - description: 'workaround for forking itself'
```

Comment:
Love to see this, did you find it via the deployment hook warning you, or did things for bad?

Reply:
Not quite sure what you mean, can you please clarify?

Reply:
Nuitka without --deployment is supposed to catch this kind of self-executing

Reply:
Ah! Right ok, I'm actually using Nuitka via `pyside6-deploy`, not sure if that makes any difference but I'll double check the arguments it's adding when I'm back next week and see if it should have caught it. I didn't see any clear warnings but again that may be due to `pyside6-deploy`.

Reply:
**Update:** I checked and the Nuitka call triggered by pyside6-deploy doesn't include `--deployment` so from what you're saying I'd expect it to have warned about the self-execution. Possibly something to do with the specific way they're launching it within `cpuinfo`?

Reply:
Ah, no, it's an untypical one, not using `-m` or `-c` but I guess, if the argument 1 is a `.py` absolute path (that does not exist), we could trigger as well, taking a note of that.